app/src/main/res/values/strings.xml

```diff
@@ -54,6 +54,7 @@
     <string name="nav_item_history">History</string>
     <string name="nav_item_settings">Settings</string>
     <string name="nav_item_alarm">Alarm</string>
+    <string name="nav_item_delete_stream_history">Delete history</string>
```

Comment:
From my POV "Delete history" is ambiguous, may something like "Clear track history" would be better.